lib/queue_bus/publishing.rb

```diff
@@ -30,6 +30,7 @@ def publish_metadata(event_type, attributes = {})
       bus_attr = { 'bus_published_at' => Time.now.to_i, 'bus_event_type' => event_type }
       bus_attr['bus_id']           = "#{Time.now.to_i}-#{generate_uuid}"
       bus_attr['bus_app_hostname'] = ::QueueBus.hostname
+      bus_attr['bus_context'] = ::QueueBus.context
```

Comment:
Hm, should we perhaps only add this attribute if `QueueBus.context` has a non-nil value in it? Seems like otherwise we'd be adding an extra attribute to *all* bus messages which seems unnecessary. Also anyone doing `attributes["bus_context"]` at the subscription level would still get the same value.